.github/workflows/python-test.yml

```diff
@@ -42,13 +42,19 @@ jobs:
         pip install --upgrade pytest-tornasync
         cd python
         pip install -e ./kserve
-        pip install -e ./aixexplainer
-        pip install -e ./alibiexplainer
-        pip install h5py==2.9.0
+        pytest ./kserve
         pip install -e ./sklearnserver
+        pytest ./sklearnserver
         pip install -e ./xgbserver
+        pytest ./xgbserver
         pip install -e ./pytorchserver
+        pytest ./pytorchserver
         pip install -e ./pmmlserver
+        pytest ./pmmlserver
         pip install -e ./lgbserver
+        pytest ./lgbserver
         pip install -e ./paddleserver[test]
-        pytest
```

Comment:
Any specific reason for not having a single `pytest` at the end?

Reply:
We are running into issues where different sub projects have dependency conflicts, so testing them separately to make sure it is testing with the right requirement versions.